### ISSUE TITLE
Adding decoding option for .ts file

### ISSUE THREAD

**Issue by Fanotec** [MedlexoNews/Update#1]
I downloaded an encoded video containing a m3u8 file and many ts files. I can play the m3u8 with vlc player. Can you support decoding and merging the ts files to mp4?

**Comment by ericsia** [comment]
> I downloaded an encoded video containing a m3u8 file and many ts files. I can play the m3u8 with vlc player. Can you support decoding and merging the ts files to mp4?

Yes it support to download m3u8, click "Combine" bar and paste the m3u8 link @Fanotec
